yum.sh

```diff
@@ -6,7 +6,26 @@
 # 
 # Based on apt.sh by Ben Kochie <superq@gmail.com>
 
-upgrades=$(/usr/bin/yum -q check-update | awk 'BEGIN { mute=1 } /Obsoleting Packages/ { mute=0 } mute && /^[[:print:]]+\.[[:print:]]+/ { print $3 }' | sort | uniq -c | awk '{print "yum_upgrades_pending{origin=\""$2"\"} "$1}')
+set -u -o pipefail
+
+filter_awk_script="$(cat << 'AWKSCRIPT'
+BEGIN { mute=1 }
+/Obsoleting Packages/ { mute=0 } mute &&
```

Comment:
move 'mute &&' to the next line since it's the two conditions together which are applied:

```
mute && /pattern/ { do stuff }
```